backend/src/main/java/atlantbh/restaurants/models/DefaultError.java

```diff
@@ -0,0 +1,28 @@
+package atlantbh.restaurants.models;
+
+public class DefaultError {
+    public String getField() {
+        return field;
+    }
+
+    public void setField(String field) {
+        this.field = field;
+    }
+
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    private String field;
```

Comment:
always put properties on top of class, then constructor, then methods.

Reply:
Removed the class in 92c4f7264b274d7da369717e0ed1478a93d30ee4 , now exceptions are handled with the built-in Exception classes